docs/pages/combobox.js

```diff
@@ -56,8 +56,135 @@ function ComboBoxExample(props) {
 }
 `}
       />
-      <GeneratedPropTable generatedDocGen={generatedDocGen} />
-
+      <PropTable
```

Comment:
This looks like a regression…I think you need to rebase to pull in Alberto's recent changes to this file